src/app/interfaces/strongRoute.ts

```diff
@@ -76,10 +76,18 @@ export class StrongRoute {
    * Add a child route
    * @param name Route name
    * @param config Additional router configurations
-   * @param isRoot Is this a root category
    */
-  add(name: string, config: Partial<Route> = {}, isRoot?: boolean) {
-    return new StrongRoute(this, name, config, isRoot);
+  add(name: string, config: Partial<Route> = {}) {
+    return new StrongRoute(this, name, config);
+  }
+
+  /**
+   * Add a child route
+   * @param name Route name
+   * @param config Additional router configurations
+   */
+  addFeatureModule(name: string, config: Partial<Route> = {}) {
```

Comment:
As per https://github.com/QutEcoacoustics/workbench-client/issues/159#issuecomment-603528912 I'd prefer if mature models used a different flag and did not reuse the `isRoot` property.

This can be fixed in a later commit, however.